tools/scripts/jira-updater/update-jira.mjs

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright © 2025 Unicode, Inc.
```

Comment:
Most source files don't have copyright notices, and that's advantageous in terms of not needing to update them or make them consistent. Not clear why this file would be exceptional. Very minor, though.

Reply:
They actually should- and our XML files do contain notices.

This is a separate part of the tooling. while I wrote it, it's possible someone will stumble upon it looking for Jira integration code.